src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#"
+    class="form form-large"
+    data-qa="big keypress"
+    >
```

Comment:
Checklist point number 5.
Also, review all the following similar cases and correct them. Your code style is a very important thing for future teamwork on the project